src/python/include/Persistent_cohomology_interface.h

```diff
@@ -73,6 +73,60 @@ persistent_cohomology::Persistent_cohomology<FilteredComplex, persistent_cohomol
     return persistence;
   }
 
+  void top_dimensional_cofaces(std::vector<int> & cofaces, int splx){
+    if (stptr_->dimension(stptr_->simplex(splx)) == stptr_->dimension()){cofaces.push_back(stptr_->simplex(splx));}
```

Comment:
Can you explain why you use `key` and `simplex`?

Reply:
No reason. Reverted to simplex handles.